test/e2e-seed-data-framework/connections.js

```diff
@@ -0,0 +1,20 @@
+import { Sequelize } from 'sequelize'
+import '@dotenvx/dotenvx'
+
+export const serviceDirectoryDb = buildSequelizeConnection(
+  process.env.DB_SD_HOST, process.env.DB_SD_NAME, process.env.DB_SD_USER, process.env.DB_SD_PASSWORD)
+
+export const referralDb = buildSequelizeConnection(
+  process.env.DB_REFERRAL_HOST, process.env.DB_REFERRAL_NAME, process.env.DB_REFERRAL_USER, process.env.DB_REFERRAL_PASSWORD)
+
+function buildSequelizeConnection (dbHost, dbName, dbUser, dbPassword) {
+  console.log(`Building DB connection for ${dbName} @ ${dbHost} (user ${dbUser})`)
+  return new Sequelize(dbName, dbUser, dbPassword, {
```

Comment:
Do we want to test the connection by calling sequelize.authenticate(); before returning? Quick fail check.

From https://sequelize.org/docs/v6/getting-started/

Reply:
Good shout. Will call `sequelize.authenticate();`

Reply:
Resolved: [0b20858](https://github.com/DFE-Digital/fh-services/pull/171/commits/0b20858b688f921717d18799ba4f0bc1f84ae19b)